Drivers/OptionRomPkg/RenesasFirmwarePD720202/README.md

```diff
@@ -0,0 +1,54 @@
+# RenesasFirmwarePD720202
+
+RenesasFirmwarePD720202 is a firmware loader for Renesas
+uPD720201/uPD720202 family USB 3.0 controllers.
+
+It provides the functionality to upload the firmware required for
+certain extension cards before they can used.
+
+## Usage
+
+ * First, a firmware image for the chipset is required.
+   The final release 2.0.2.6 (K2026090.mem) can be downloaded from
+[https://www.renesas.com/en/products/interface/usb-switches-hubs/upd720202-usb-30-host-controller#documents](https://www.renesas.com/en/products/interface/usb-switches-hubs/upd720202-usb-30-host-controller#documents)
+   after registering for a free account.
+ * The firmware image is uploaded into the chipset.
+   This process is non-persistent, the chipset RAM is cleared when the power supply is removed.
+
+To use this driver, update the platform .dsc file:
+```
+  #
+  # Renesas PD720202 XHCI firmware uploader
+  #
+  Drivers/OptionRomPkg/RenesasFirmwarePD720202/RenesasFirmwarePD720202.inf {
+    <LibraryClasses>
```

Comment:
Hmm...
Doesn't any real platform need to define a global resolution for DxeServicesLib anyway, since it's a dependency for DxeMain?

Reply:
Oh, good point. I'll remove that from the README.